api/internal/plugins/execplugin/execplugin.go

```diff
@@ -50,7 +51,9 @@ func (p *ExecPlugin) ErrIfNotExecutable() error {
 	if err != nil {
 		return err
 	}
-	if f.Mode()&0111 == 0000 {
+	// In Windows, it is not possible to determine whether a
+	// file is executable through file mode.
+	if f.Mode()&0111 == 0000 && runtime.GOOS != "windows" {
```

Comment:
Can you add a test for this behavior?

Reply:
@Shell32-Natsu Thank you for the review. I'm not good at Go development, but I tried to add a test. I tried to skip the test in Windows environment, what do you think?

Reply:
@gochist To line 55, can you add this:

// TODO: provide for setting the executable bit on Windows.
// The (fs *fileStat) Mode() (m FileMode) {} function in
// in https://golang.org/src/os/types_windows.go
// lacks the ability to set the executable bit.

Reply:
then i think we should pull it in, and unblock windows use.

Reply:
@monopole Thank you. I've addressed your review. And I will open an issue in the golang repo within a few days.

Reply:
@monopole I've opened an issue, https://github.com/golang/go/issues/41809 in the golang repo.